### ISSUE TITLE
modal does not shows up

### ISSUE THREAD

**Issue by neo-xy** [shlomiassaf/ngx-modialog#427]
took the example 

```
modal.alert()
    .title('Hello World')
    .body('In Angular')
    .open();
```

but no modal is showing up (no errors) any idea what can be the reason ? (Angular: 5.0.3)

**Comment by worldbeater** [comment]
The same issue. Used instructions from readme, plugin hides the scroll bar but does not show any dialogs. So the plugin seems to be broken or outdated and can not be used with Angular 5 right now, need a fix

**Comment by qqnc** [comment]
Do you wanna install a minimum project and check? and also check the version of your bootstrap package.  See if it's v3 or v4

**Comment by worldbeater** [comment]
>If it's v4. Add this in your component in this case.

This should definitely be added to project's readme file to avoid such inconveniences. Thanks a lot for sharing a solution, @qqnc!

**Comment by qqnc** [comment]
I think that since now bootstrap v4 has been official. Probably better make bootstrap v4 enabled in default, and some documentations for bootstrap v3.

**Comment by shlomiassaf** [comment]
Hi guys,

PR's are welcome.

I suggest waiting for my upgrade to V6, then if someone will post a PR that will be great.

Don't have time to manage all issues.

**Comment by qqnc** [comment]
Thanks, @shlomiassaf 

We all understand your hard working on this project.  We'll wait for your upgrade to V6, and see what we can help for the issues.  Do you agree to switch to bootstrap 4 as default, and documents for v3?

**Comment by zios07** [comment]
@qqnc  Thanks ! Your answer helped me.

**Comment by dbusacca** [comment]
Now I've got this issue with version 6.0.0 and boostrap 3

**Comment by shlomiassaf** [comment]
closed by #434
